refapp/SKWebApi/Config/AIServiceConfig.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// TODO: align with SK naming and expand to have all fields from both AzureOpenAIConfig and OpenAIConfig
+
+namespace SemanticKernel.Service.Config;
+
+#pragma warning disable CA1812 // Avoid uninstantiated internal classes - Instanciated by deserialing JSON
+internal class AIServiceConfig
```

Comment:
Mental note to split configurations for each type of backend. OI and AOI are different enough to be drifting with each api revision. No need to change anything for this PR.

Reply:
Noted in comment